### ISSUE TITLE
Rasberry Pi 4 -  ERROR help? - No module named 'click' Install the missing module noted above.

### ISSUE THREAD

**Issue by aliyousu** [Hari-Nagarajan/fairgame#604]
Hello, 

I had the bot running but i updated the bot today and keep running in the following error. 

"    FairGame Copyright (C) 2021 Hari Nagarajan
        This program comes with ABSOLUTELY NO WARRANTY; for details
        start the program with the `show --w' option.

        This is free software, and you are welcome to redistribute it
        under certain conditions; for details start the program with
        the `show --c' option.

No module named 'click'
Install the missing module noted above." 

Before anyone asks i did run the following before trying to run the bot:    

pip3 install pipenv
export PATH=$PATH:/home/$USER/.local/bin
pipenv shell 
pipenv install

Would appreciate you help!!

**Comment by DakkJaniels** [comment]
before you tried running, as in, after you updated the bot?

**Comment by aliyousu** [comment]
Yes i updated it - redownloaded the files and getting this error. Any suggestions?

**Comment by DakkJaniels** [comment]
Sorry, I mean, you did all that stuff you wrote in the end after you updated the files? 
```pipenv shell
pipenv install```

**Comment by aliyousu** [comment]
Yup i completed those steps. I am going to run the install from the beginning and hopefully that corrects this issue if not i ll report back

**Comment by DakkJaniels** [comment]
so you did the following?

1. downloaded files
2. ran pipenv shell/pipenv install stuff
3. downloaded new files
4. ran pipenv/pipenv install

**Comment by aliyousu** [comment]
Hey DakkJaniels, yes i followed those steps it wasnt working. 

I deleted the fairgame folder and reinstalled it. Then i ran it with "python app.py amazon" it was still giving me the same error:  No module named 'click'
Install the missing module noted above."

I then tried the following "pipenv run python app.py amazon" and it ran properly without issues. I am not sure why the normal run command wasnt working, if you have any suggestions let me know. 

Regards,
Yousuf
